src/overview/enhancer.js

```diff
@@ -61,8 +61,12 @@ const locationSync = ReduxQuerySync.enhancer({
 const hydrateStateFromStorage = store => {
     const hydrate = (key, action) =>
         browser.storage.local.get(key).then(data => {
-            if (!data[key]) return
-            store.dispatch(action(data[key]))
+            if (key === constants.SHOW_TOOL_TIP) {
```

Comment:
now that `setShowTooltip` has a common type signature, this condition shouldn't need to be here anymore, right?

Reply:
There is a need to do this because as we know we have to set when the value is false, but in other cases, we are just leaving( I mean we are set the value when there is a false value).

Reply:
Kinda see what you mean, but I still see no reason for this condition; it only needs to rehydrate the state if the stored value is `true`, as the only other case of `false` is covered by default state. 

But this brought up a good point with the existing condition: instead of `!data[key]`, which will pass if the stored value is falsey, including when nothing set, we should change it to `data[key] == null` which will only cover the case of not being set, but let stored falsey values be rehydrated (like `false`, in this case). Do you see any issues with that?